code/modules/projectiles/guncrafting/handle.dm

```diff
@@ -0,0 +1,23 @@
+/obj/item/weapon/gun_attachment/handle
+	name = "base"
+	desc = "sfdahgfhgasfhgiahjfhajsdjkJA"
```

Comment:
My daddy taught me a few things too, like, uh, how not to trigger maintainers by using someone else's idea.

>The handle for a gun.